decimal64.go

```diff
@@ -242,40 +240,9 @@ func (d Decimal64) parts() (fl flavor, sign int, exp int, significand uint64) {
 }
 
 // getParts gets the parts and returns in decParts stuct, doesn't get the magnitude due to performance issues\
-// TODO: rename this to parts when parts is depreciated
 func (d Decimal64) getParts() decParts {
-	var fl flavor
-	var sign, exp int
-	var significand uint64
-	sign = int(d.bits >> 63)
-
-	switch (d.bits >> (63 - 4)) & 0xf {
-	case 15:
-		switch (d.bits >> (63 - 6)) & 3 {
-		case 0, 1:
-			fl = flInf
-		case 2:
-			fl = flQNaN
-		case 3:
-			fl = flSNaN
-		}
-	case 12, 13, 14:
-		// s 11EEeeeeeeee (100)t tttttttttt tttttttttt tttttttttt tttttttttt tttttttttt
-		//     EE ∈ {00, 01, 10}
-		fl = flNormal
-		exp = int((d.bits>>(63-12))&(1<<10-1)) - expOffset
-		significand = d.bits&(1<<51-1) | (1 << 53)
-	default:
-		// s EEeeeeeeee   (0)ttt tttttttttt tttttttttt tttttttttt tttttttttt tttttttttt
-		//   EE ∈ {00, 01, 10}
-		fl = flNormal
-		exp = int((d.bits>>(63-10))&(1<<10-1)) - expOffset
-		significand = d.bits & (1<<53 - 1)
-		if significand == 0 {
-			exp = 0
-		}
-	}
-	return decParts{fl, sign, exp, uint128T{significand, 0}, 0, &d}
+	flavor, sign, exp, significand := d.parts()
```

Comment:
This is repeated code and now just calls the other parts method
On second thought, `parts()` will not be depreciated as there are some functions which do not need a `decParts `struct